plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -326,6 +326,20 @@ public synchronized List<String> getAllViews(String databaseName)
         return tables.build();
     }
 
+    @Override
+    public synchronized Optional<List<SchemaTableName>> getAllTables()
+    {
+        return Optional.of(this.relations.keySet().stream()
```

Comment:
nit: Optional.of(ImmutableList.copyOf(relations.keySet()))` and for view also

Reply:
Done